src/PantherTestCase.php

```diff
@@ -23,6 +27,39 @@ abstract class PantherTestCase extends WebTestCase
 
         public const CHROME = 'chrome';
         public const FIREFOX = 'firefox';
+        public const COVERAGE_DIRECTORY = __DIR__.'/../var/panther-coverage';
+
+        public function run(TestResult $result = null): TestResult
```

Comment:
This will not work anymore with PHPUnit 10 ;(
Now it is `final public function run(): void`

If someone found a solution to get it work with PHPUnit 10, please let us know.